qiskit/providers/basicaer/qasm_simulator.py

```diff
@@ -662,10 +662,10 @@ def _validate(self, qobj):
             name = experiment.header.name
             if experiment.config.memory_slots == 0:
                 logger.warning(
-                    'No classical registers in circuit "%s", ' "counts will be empty.", name
+                    'No classical registers in circuit "%s", counts will be empty.', name
```

Comment:
I had assumed you'd just done these replacements by regex or something, but then this one really threw me! Did you still have an automated way of doing this by AST or something?

Reply:
The stdlib AST already merges strings, so I made a dumb hack around tokenize https://docs.python.org/3/library/tokenize.html

I did do some manual review as well, but as you point out I missed quite a few things.

Reply:
Oh, that's really clever.

I wouldn't say they were things that were missed (I think there were some %-strings which could be f-strings, but those don't really matter), just a couple of pre-existing typos, really.